portal/client/src/services/user.service.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import authHeader from './auth.header';
+import { API_BASE_URL } from '../constants/apiConstants';
+
+// Data service
+const getStudyData = () => {
+    return axios.get(API_BASE_URL + "/user", { headers: authHeader() });
```

Comment:
recommend using `/studies` and the whole service also as study.service or something.